src/app/components/send-coins-form/send-coins-form.component.ts

```diff
@@ -218,4 +218,12 @@ export class SendCoinsFormComponent implements OnInit, OnDestroy {
     }
     return true
   }
+
+  formatCurrData(coins) {
+    let formattedCoins = []
```

Comment:
i dont think you need to use let here since you're not reassigning the variable, only pushing to it.

Reply:
try const.

Reply:
ah right. Brain thought "changing = let". Will remember "redefine = let"
Updating now